### PR TITLE
FW-1976 Chat widget setting for square icon

### PR DESCRIPTION
**Pull request by harishbala1o**
### What do you want to achieve?
- Wanted to change the chat widget icon shape to a square by adding a parameter in widget settings.

### PR Checklist

- [X] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?
- Made the code changes and tested the same by updating Kommunicate settings in the script.
- `Note:` The image you are using for the icon needs to be in a square avatar 

`(function(d, m){
        var kommunicateSettings = 
            {"appId":"307ec0c522a953b521d67a195c64ade3","popupWidget":true,"automaticChatOpenOnNavigation":true,"iconShape":"square"};
        var s = document.createElement("script"); s.type = "text/javascript"; s.async = true;
        s.src = "http://localhost:3030/v2/kommunicate.app";
        var h = document.getElementsByTagName("head")[0]; h.appendChild(s);
        window.kommunicate = m; m._globals = kommunicateSettings;
    })(document, window.kommunicate || {});`
    
    
closed
---
open
---
sqreimage

### What new thing you came across while writing this code? 

- Creating variables with kommunicate._globals

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Comment by harishbala1o** [comment]
Expected output

square icon

**Review by Prochnost55 — DISMISSED** [review]

**Review by Ojashwa — COMMENTED** [review]
what if we use a custom image in the icon?
also, check if it is working fine with the greeting message.

**Review by Prochnost55 — CHANGES_REQUESTED** [review]
Used a custom image for chat widget icon and also toggled the widget icon to left and right position. 
The UI seems to be bit off.
Attaching SS for ref.

Screenshot 2021-05-05 at 10 23 55 AM
Screenshot 2021-05-05 at 10 23 26 AM
Screenshot 2021-05-05 at 10 25 42 AM

File: webplugin/js/app/kommunicate.custom.theme.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ function KmCustomTheme() {
                 WIDGET_SETTINGS && WIDGET_SETTINGS.secondaryColor
                     ? WIDGET_SETTINGS.secondaryColor
                     : DEFAULT_SECONDARY_BACKGROUND_COLOR;
+            var squareIcon = kommunicate._globals.iconShape == 'square' ? '.km-chat-icon-sidebox { border-radius: 10% !important; box-shadow: 0 3px 4px 1px rgb(107 104 104 / 50%) !important; } .chat-popup-widget-container--horizontal {border-radius: 12px !important;}' : null;
```
Comment (devashishmamgain) [inline webplugin/js/app/kommunicate.custom.theme.js:36]: move css to CSS file and only css class name to the element.

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

**Review by devashishmamgain — APPROVED** [review]

File: webplugin/js/app/kommunicate.custom.theme.js (hunk 1 above)
Comment (harishbala1o) [reply webplugin/js/app/kommunicate.custom.theme.js]: Updated the CSS changes

**Review by Ojashwa — APPROVED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Review by Prochnost55 — APPROVED** [review]

**Pull request merged**